bentoml/_internal/frameworks/transformers.py

```diff
@@ -106,427 +65,209 @@ def check_flax_supported() -> None:  # pragma: no cover
-                revision=self._revision,
-                use_fast=self._use_fast,
-                use_auth_token=self._use_auth_token,
-                model_kwargs=self._model_kwargs,
-                device=self._device,
-                **self._kwargs,
-            )
-
-    def _run_batch(self, *args: t.Any, **kwargs: t.Any) -> t.Any:
-        return self._pipeline(*args, **kwargs)  # type: ignore
+                # assign CPU resources
+                kwargs = {}
 
+            self.pipeline = load_model(bento_model, **kwargs)
 
-def load_runner(
-    tag: t.Union[str, Tag],
-    *,
-    tasks: str,
-    framework: str = "pt",
-    device: int = -1,
-    name: t.Optional[str] = None,
-    **pipeline_kwargs: t.Any,
-) -> "_TransformersRunner":
-    """
-    Runner represents a unit of serving logic that can be scaled horizontally to
-    maximize throughput. :func:`~bentoml.transformers.load_runner` implements a Runner class
-    that wrap around a transformers pipeline, which optimize it for the BentoML runtime.
-
-    .. warning::
-       :func:`load_runner` will try to load the model from given :obj:`tag`. If the model does not
-       exists, then BentoML will fallback to initialize pipelines from transformers,
-       thus files will be loaded from huggingface cache.
-
-    Args:
-        tag (:code:`Union[str, Tag]`):
-            Tag of a saved model in BentoML local modelstore.
-        tasks (:code:`str`):
-            Given tasks for pipeline. Refers to `Task Summary <https://huggingface.co/transformers/task_summary.html>`_
-            for more information.
-        framework (:code:`str`, default to :code:`pt`):
-            Given frameworks supported by transformers: PyTorch, Tensorflow
-        device (`int`, `optional`, default to :code:`-1`):
-            Default GPU devices to be used by runner.
-        **pipeline_kwargs(`Any`):
-            Refers to `Pipeline Docs <https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.pipeline>`_ for more information
-            on :obj:`kwargs` that is applicable for your specific pipeline.
+            self.predict_fns: dict[str, t.Callable[..., t.Any]] = {}
+            for method_name in bento_model.info.signatures:
+                self.predict_fns[method_name] = getattr(self.pipeline, method_name)
 
-    Returns:
-        :obj:`~bentoml._internal.runner.Runner`: Runner instances for :mod:`bentoml.transformers` model
+    for method_name, options in bento_model.info.signatures.items():
 
-    Examples:
+        def _run(self: TransformersRunnable, *args: t.Any, **kwargs: t.Any) -> t.Any:
+            return getattr(self.pipeline, method_name)(*args, **kwargs)
 
-    .. code-block:: python
+        TransformersRunnable.add_method(
+            _run,
+            name=method_name,
+            batchable=options.batchable,
+            batch_dim=options.batch_dim,
+            input_spec=options.input_spec,
+            output_spec=options.output_spec,
```

Comment:
refers to https://github.com/bentoml/BentoML/blob/runners-1.0/bentoml/_internal/frameworks/FRAMEWORK_TEMPLATE_PY for bug fix

Reply:
or refers to the sklearn one: https://github.com/bentoml/BentoML/blob/runners-1.0/bentoml/_internal/frameworks/sklearn.py#L185